### ISSUE TITLE
CRM (Customer Relationship Management) Implementation

### ISSUE THREAD

**Issue by anazariz** [lzim/teampsd#1299]
**Sidebar Clicks** 

i. **Pipeline** - Open Features (Will stay here until card is closed)   
ii. **Label**- assign workgroup (lead) responsible for tracking the task  
iii. **Projects** - Feature Tracker (work_breakdown)  
iv. **Milestones** - select monthly sprint milestone within the workgroup   
v. **Estimate** - Estimate feature in complexity points   
vi. **Epics** - assign to team-wide epic and TeamPSD master plan project  
vii. **Releases** - assign to key team release grouped with other relevant issues  

*delete the above after completing*

**1. Feature Description: What is the goal of this feature and why are we working on it?** 
------------
The CRM (Customer Relationship Management) folder in TeamPSD documents the MTL team conducted a series of market research activities in 2018 and 2019 in search of a suitable platform capable of handling the support and informational needs of the MTL platform’s user communities. While CRM covers many aspects of Customer Relationship Management, the feature that was immediately needed upon go-live was the effective management of the inbound emails from customers requesting help and information for which two Microsoft Exchange shared accounts "MTL HELP" and "MTL INFO" were created. As the documents show, Microsoft Dynamics was selected because of its native integration with the Microsoft Outlook platform. As the project progressed, it was decided that a critical component of the MS Dynamics platform, the effective management of responses to inbound emails, had to be implemented prior to the implementation of other CRM capabilities. Hence, the project was divided into two phases. 
<ul>
<li>Phase I - Capability to effectively manage inbound emails to “MTL HELP” and “MTL INFO” as well as responses to those emails by designated responders.</li>
<li>Phase II - Implementation of additional CRM capabilities as needed.</li>
</ul>

Example:

====== here where you describe the requirements

**Want to create a Community of Practice for the users of the SIM UI to access in order to have a forum for discussion and to ask/answer questions** 

 
**2. Requirements, Design, and Test Cases: Outline of the necessary requirements, design, and test steps**  
<details>
<summary> Requirement Helper </summary>
 1. Requirement Definition (What & Why)
 </br></br>
2. Verification Method (How)

<strong>Documentation of requirements</strong> - Review of definition of the requirements.  

<strong>Analysis</strong> - Code, data, model reviewed. 

<strong>Test</strong> - Procedures must be well documented.  

<strong>Inspect</strong> - Purpose, methods and procedures must be well documented so result can be duplicated using the same inspection procedures. May include inspection of design or an algorthm (aka process map of process).

<strong>Demonstrate</strong> - A comprehensive demonstration and/or application of the functions and features of the system or its components.  

<strong>Infer</strong> - from “Why”/purpose of requirement definition.  
</br></br>
3. Reviewer (Who)

Within Workgroup Member  

Verify across workgroups  

Required or Optional?  
</br> </br>
4. Location in Tracker
</details>

-------------

- [ ] Requirement 1 Met
<details open=true>
<summary>Requirement 1</summary>

  1. Step 1 Requirement Definition (What & Why)   
  Example:  
  **Community Button links out of Sim UI to the GitHub Community of Practice.**

======here 2=====actual requirement 1
2 and 3 should be filled
  
  2. Verification Method (How)   
  Example:  
  - [X] Documentation of requirements - Reviewed definition of the requirements at **sim_UI_workgroup** on 4/07/2020.  
  3. Reviewer (Who & When)  

  Example:  
  - [x] Verified design with Lindsey & Stacey at **sim_UI_workgroup** on 04/07/2020.
  4. Location in Tracker  
  *Upon Approval Card can move to operations/research_to_do*

  ---
  1. Step 2 Design Definition (What & Why)  
   Example:  
  **Create Community Button that links out of Sim UI to the GitHub Community of Practice through 

===== herer ====== design

a new "I have a question" ["Community   Button" > "I have a question" Button > GitHub "Community of Practice"].**
  2. Verification Method (How)  
   Example:  
  - [ ] Design document of the 3 buttons within the Button - Review this design at **sim_UI_workgroup** on 4/14/2020.
  3. Reviewer (Who & When)  
   Example:    
  - [ ] Verified design prototype with Lindsey & Stacey at **sim_UI_workgroup** on 04/14/2020.
  4. Location in Tracker  
  _After approval card can move to under_development once work has begun._

  ---
  1. Step 3 Build/Test Definition (What & Why)  
  Example:  
  **Implement the new buttons on the SIM UI**

====== here build
 
  2. Verification Method (How)  
  Example:  
  - [X] Analysis - Code tested by Anthony on 04/18/2020.  
  - [ ] Demonstration of the new design/functionality on the platform - Review this demo at **sim_UI_workgroup** on 4/21/2020.  
  3. Reviewer (Who & When)    
   Example:  
  - [X] Anthony ran test scripts outlined in Design Document on 04/18/2020.
  - [ ] Verified demo with Lindsey & Stacey at **sim_UI_workgroup** on 04/21/2020.
  4. Location in Tracker  
  _Once Review has begun Card should either move to functional_testing or measurement based on the test being done._
</details>

<button type=button onclick=print(Content) > Add a requirement button </button>

**3. Review**: 
----------------

**Comment by lzim** [comment]
**Decided at #manuscriptmonday that Anthony will create this new split off Issue from Issue #1299**

- Create a new issue card in May Priority Epic and walkthrough the rules creation process with
- Stacey will be lead  provide feedback and Define SOP for how we define the

a) inbound rules
b) outbound rules
c) We will ALL have full team visibility on email inbox/outbox team sub-folders, but use of Outlook Rules Wizard provides no additional reporting functions as a back-stop against slow responses, missed responses, teams that are particularly active etc. (cross-ref #1023.

**Comment by lzim** [comment]
Questions from the #manuscriptmonday meeting 4/13/2020

Jane's questions 

Is it possible to set up a rule in outlook that makes sure the FROM address is by default help@mtl.how  when it's to an email in that account? If not, what are the ramifications if I forget to change the response email address?
The flip of that is the risk of my sending emails that should be from my VA address that look like they're from help@mtl.
Can we think of a way to "call" or "assign" who will respond to a given question? May just be a workflow understanding, e.g. I will forward an email to Gayle, my co-fac, if I see it first and think she'd be the better one to respond; or to Jen if I'd like to ask her to respond. "Calling it" would be to avoid two people simultaneously working on responses and the learner getting 2 responses.

- We should document the questions asked so we can create standard responses over time

**Comment by branscombj** [comment]
> - We should document the questions asked so we can create standard responses over time

Not sure how heavy the flow of questions will be, but this could be a ton of work. If it's a manageable amount, seems like a task for HQ Team Supports to code incomings and compile the responses that have been sent on similar questions; then the full Co-Facs team can decide if we want to standardize a response.

We may also want to have a practice of checking the COP before tailoring a response if the question is already answered. A personal reply would still be good, but it could just link the learner to the answer in mtl.how issues (meanwhile elevating the visibility of that resource).

**Comment by dlkibbe** [comment]
@branscombj I actually deleted the question because I recall @anazariz saying on our "orientation" call that there would be something in the spreadsheet or rules that would look for key words.
@anazariz  - Ash, is that the case?

**Comment by branscombj** [comment]
> 3. Can we think of a way to "call" or "assign" who will respond to a given question? 

One way to "call" it would be to use the Flag option in Outlook. When I see it flagged, I'll know someone is on it. If I don't see it unflagged within __, I can check in with my co-fac/hqts. 

-> Question: When we're sharing an email account, will it be apparent which one of us created the flag?

I didn't know this until now, but if you right-click the flag, you can specify when you're going to respond by and generate a reminder for  yourself. 

-> Question: Will that reminder go to everyone who shares the account in Outlook, or only to the person who flagged it? If the former, ditch that idea!



![image](https://user-images.githubusercontent.com/29778290/79161173-041a5100-7da9-11ea-9943-66ed7e14976b.png)

![image](https://user-images.githubusercontent.com/29778290/79161496-928ed280-7da9-11ea-8a88-b8db2d8de522.png)


Another way - probably better - would be to use the Categorize option. This puts a color-coded box on the email and we can assign names to the categories. This would stay with the email forever once it's handled, so we'll know who responded to what. Here, I've put some category markers on my VA emails as an example. The pink one is one I had named Admin, so you see the big pink bar labeled Admin above the email preview as well. This would say Lindsey, David, Rita, etc.

![image](https://user-images.githubusercontent.com/29778290/79162182-d46c4880-7daa-11ea-9c68-b631d388aa52.png)

**Comment by branscombj** [comment]
> @branscombj I actually deleted the question because I recall @anazariz saying on our "orientation" call that there would be something in the spreadsheet or rules that would look for key words.
> @anazariz - Ash, is that the case?

Ahh - good question, @dlkibbe  - Since Ash has moved to using built-in Outlook functionality to do the sorting and tracking rather than the spreadsheet, I'm not sure. It's definitely possible to search through a specific (or set of) email folders for keywords as one way to more quickly do the hand-sorting.

**Comment by anazariz** [comment]
Sorry I was working on a different item. Please let me absorb the feedback and respond to the questions shortly. Thanks.

**Comment by anazariz** [comment]
@lzim @dlkibbe  @branscombj @anthonycpichardo @staceypark  
Is it possible to set up a rule in outlook that makes sure the FROM address is by default help@mtl.how  when it's to an email in that account? If not, what are the ramifications if I forget to change the response email address?

Ash: When you start from a mailbox by default it becomes the "From". Please make sure that you have the From field turned on just to be sure. In this example I started from a personal mailbox,  which is only partially shown here.
  
![image](https://user-images.githubusercontent.com/39805164/79163753-82202d00-7d94-11ea-8702-854235213ca4.png)

The flip of that is the risk of my sending emails that should be from my VA address that look like they're from help@---. 

ASH: I think if you remember to start from the help Account, you should be fine. All you have to do is click that account (which effectively switches you to it). 

Can we think of a way to "call" or "assign" who will respond to a given question? May just be a workflow understanding, e.g. I will forward an email to Gayle, my co-fac, if I see it first and think she'd be the better one to respond; or to Jen if I'd like to ask her to respond. "Calling it" would be to avoid two people simultaneously working on responses and the learner getting 2 responses.

ASH: This in fact is a workflow item. As I mentioned during my last walk-through of "Automation Engine 4" (a few weeks ago) there are extra user-define fields (e.g. Category), which can be used for these kinds of assignments.   

Is it possible for people to mark that they will respond to a message and have a notification to check if something hasnt been responded to?

ASH: Use a user defined field (see above). There are views in outlook that will allow you to customize the fields you see. What you see on the screen is what Microsoft starts with initially. Not sure how many user-defined field are in there. But I'll check.  

Can these rules be integrated with any type of dashboarding that would let us catch missed emails? 
Answer: Yes we can do reporting through Excel

ASH: Yes as I mentioned during our walk-through, I will change the "Automation Engine 4", which we walked through last time around to get this done. Excel is not the most efficient but it should give us the minimal amount of reporting we need.   

If we forward an email does it show up as though it's been responded to?
Answer: We can send a copy of the email to responders email address or potentially send a notification

**Comment by anazariz** [comment]
@branscombj I actually deleted the question because I recall @anazariz saying on our "orientation" call that there would be something in the spreadsheet or rules that would look for key words.
@anazariz - Ash, is that the case?

ASH: Yes we should be able to bring emails into Excel for analysis. I would not recommend Excel for real-time work (intercepting inbound/outbound emails). Because Exchange rules, which we walked through today would do a much better job there. But for reporting and analysis Excel may do OK.

**Comment by anazariz** [comment]
One way to "call" it would be to use the Flag option in Outlook. When I see it flagged, I'll know someone is on it. If I don't see it unflagged within __, I can check in with my co-fac/hqts.

ASH-Marking inbound emails with Flags can also be automated with the rules engine. Also, as mentioned above, we can use the user-define fields (they're not in the default views that Outlook comes with).    

-> Question: When we're sharing an email account, will it be apparent which one of us created the flag?

ASH: Not sure. But I would think so because these are shared accounts (created for that purpose). Would be easy to check. 

I didn't know this until now, but if you right-click the flag, you can specify when you're going to respond by and generate a reminder for yourself.

-> Question: Will that reminder go to everyone who shares the account in Outlook, or only to the person who flagged it? If the former, ditch that idea!

ASH-sorry not sure. But we should be able to test these out.

**Comment by anazariz** [comment]
Ahh - good question, @dlkibbe - Since Ash has moved to using built-in Outlook functionality to do the sorting and tracking rather than the spreadsheet, I'm not sure. It's definitely possible to search through a specific (or set of) email folders for keywords as one way to more quickly do the hand-sorting.

ASH: The rules can be complex. You could force an email to be routed based on it's TO: field as well as keywords. You can also route it to a Sub-Folder within Sub-Folder......e.g. Group1/Technical Group1/Other ...etc. However, if we want to do that, I'd have to know to so that I can put recursive logic in the reporting code to go thru all the folders and their sub-folders.

**Comment by anazariz** [comment]
@lzim @dlkibbe @branscombj @anthonycpichardo @staceypark

[This tutorial](https://www.slipstick.com/tutorial/create-a-custom-field-to-mark-messages/) shows how to create user defined fields in Outlook. In fact, the example talks about how a new field could be used to keep track of emails that have not received a response. Second example talks about a notes field, which may also be useful. Please ignore the part that talks about automation via VBA coding. My experience has been that VBA code place in Outlook is deleted by the nightly cleanup process.

**Comment by anazariz** [comment]
Thanks @anthonycpichardo. 

@lzim @dlkibbe @branscombj @staceypark

I added/updated the items we covered in the following Lucid meeting:
crm, fac cybersem Actions
Monday, April 13, 9 AM - 11 AM Los Angeles

**Comment by lzim** [comment]
**Cross-ref #1023 - Bridging to CRM** 
- Call scheduled for Friday 4/16 at 11AM w/Enterprise Cloud Computing

**Comment by staceypark** [comment]
**Summary from Friday 4/16 at 11AM w/Enterprise Cloud Computing**
We decided **not** to retire the current instances, but instead Reyes will try to pursue continued funding for the instances (support for current license instances cost $15000). With Ash on board, we think it will be useful to delegate Ash to explore/develop in the environment and take this further. In another year, at the next renewal of licenses, we will have a better idea of whether we want to continue pushing forward.

**Next Steps:** 
1. Reyes will advocate for funding
2. Once we have funding, Reyes will reach out to Enterprise License Agreement Contract Representative to modify contract
3. Give Ash developer access to create an environment at the application level
4. Follow VA compliance and mandate requirements once we go to PROD with live VA data:
- Register system publically on VA Systems Inventory (VASI) 
- Authority to Operate (ATO)

**Comment by lijenn** [comment]
8AM WG Leads 8/10:
- Review @anazariz's prototype and #841 team_tracker due to dependencies at Support WG meeting.
- Record the meeting as a MVP prototype for people missing at the Support WG and to orient those that will need to use it and can identify pain points and feedback for it.

**Comment by anazariz** [comment]
## CRM Phase I Refresher

**Below are two methods for handling incoming emails into the MTL Help and MTL Info Shared Exchange Mailboxes will be discussed.**

Methods:
 1. Using Microsoft Outlook only
 2. Using Microsoft Excel

## Method 1 - Using Microsoft Outlook only

Both the Facilitator and the MTL staff must have access to both Shared Exchange mailboxes. This can be verified by logging into your personal  VA email. 

![image](https://user-images.githubusercontent.com/39805164/89935485-19708200-dbc7-11ea-845c-9ce21b3b9d59.png)

As the shared folders are two large buckets that will hold all incoming emails to all Facilitators and MTL staff, use the following method to group them as needed.

Right Click on the MTL Help or MTL Info inbox to create a new sub-folder. Name the sub-folder as you wish for your team. 

![image](https://user-images.githubusercontent.com/39805164/89936360-6143d900-dbc8-11ea-9e2b-3bc8c26189b6.png) 

Name the sub-folder as you wish for your team.  Do this for the inbox and outbox, as seen below.
  ![image](https://user-images.githubusercontent.com/39805164/89937042-681f1b80-dbc9-11ea-8e33-f38495669540.png)

Incoming emails (to MTL Help and MTL Info) will be routed to these folders via Outlook Rules.

![image](https://user-images.githubusercontent.com/39805164/89937674-470afa80-dbca-11ea-8e11-c0a8835a0cef.png)

If you do not see the Rules button just go to the File menu and there you'll see the "Manage Rules and Alerts" button.

![image](https://user-images.githubusercontent.com/39805164/89939316-aa962780-dbcc-11ea-9b99-419d075a5f26.png)

The objective of creating incoming and outgoing rules is to direct the respective emails to the folder(s) you just created. 

Outlook Rules are powerful. You should create separate rules for inbound and outbound mails.

- For inbound (Inbox) “From” is the list of LEARNERS – multiple LEARNERS can be selected from VA GAL (Global Address List) – this is where maintenance is required. Emails that are not moved to the folder you have created, have not been included in the “From” (inbound) rule and should be added there. The items that were not processed by the rule can either be manually moved to the right folder or they can be picked up by running the rule once they’re added.

![image](https://user-images.githubusercontent.com/39805164/89940343-33619300-dbce-11ea-9edd-3abfb44ee815.png)

![image](https://user-images.githubusercontent.com/39805164/89940393-4a07ea00-dbce-11ea-945c-742a33fecdf3.png)

 - For outbound (Sent Mail) “From” is the Help or Info account. CO-FACILITATORS & HQ SUPPORT TEAM STAFF can send emails from Help and Info account by choosing them in the right From account before sending emails. To display the **From field** click on the Options tab and select From in the Show Fields section. Now the From field will be displayed when composing a message in the future. 2) When composing a message click on the From in the drop-down box and select Other E-Mail Address.
![image](https://user-images.githubusercontent.com/39805164/89941265-9f90c680-dbcf-11ea-946c-22cd17b45e69.png)

There are neumerous internet resources for creating and managing Outlook Rules such as  [this one by Microsoft](https://support.microsoft.com/en-us/office/video-how-to-set-up-rules-in-outlook-75ab719a-2ce8-49a7-a214-6d62b67cbd41).

🔶 *Please note that all the above (moving emails to the correct folder) can be done manually as well.* 

**How to organize emails once they are in your correct Inbox/Outbox Folders.**

Outlook emails have user define fields that can be used as flags or other purposes such as "Have I responded to this email or not?" 

[This tutorial](https://www.slipstick.com/tutorial/create-a-custom-field-to-mark-messages/) shows how to create user defined fields in Outlook. In fact, the example talks about how a new field could be used to keep track of emails that have not received a response. Second example talks about a notes field, which may also be useful. Please ignore the part that talks about automation via VBA coding. My experience has been that VBA code place in Outlook is deleted by the nightly cleanup process.  

## Method 2 - Using Microsoft Excel

**Comment by lijenn** [comment]
Question/feature to bring up to Special Forces:
Could we automate pre/post emails to go out using Dynamics?
The 'Delayed Delivery' send function in Outlook itself is not really reliable/unresponsive based on how we used it with the tt-reports. 
If we could set up the pre/post emails to actually go out on time by themselves, that'd take the cognitive load off of having to remember to do it.

**Comment by lzim** [comment]
Do we have a GH issue task for @staceypark  and @lzim  (or possibly @lijenn) to review the user stories we hoped that Scott would re-send?

**Comment by lijenn** [comment]
Yes, I believe its #1497

**Comment by lzim** [comment]
Let's move this to **nov_epic** along with #1497. We have to prioritize R21, R01, IIR.

Thanks! @lijenn 

@staceypark @anthonycpichardo I think this is likely really helpful information for the user stories workflow for Scott.

**Comment by lijenn** [comment]
We are now being supported by Liberty IT, configurations that need to be set up:
Story 6 roadmap: https://lucid.app/lucidchart/4fb06b59-38f5-4d61-86f0-4620ceb93196/edit?page=0_0#

Configurations below in both lists need to be completed before may_epic:
- [ ] #1728 - Data Dictionary (All)
- [ ] #1885 - Exchange and Outlook (Ash) 
- [ ] #1729 Campaigns against 3 MTL Phases (James and Matt)
- [ ] #1727 CRM Templates for pre/post email (Jane)
- [ ]    #1726 Import Mapping of Teams Facilitator Asignment (Jenn)

   Needs assignment:
- [ ]     #1898 File Storage/Access facilitate
- [ ] #1897 Documentation/Notes facilitate
- [ ] #1896 10 Standard CRM & co-plan Dashboards
- [ ] #1895 Facilitator Assignment facilitate
- [ ] #1894 Outlook Integrations of Invitations facilitate
- [ ] #1893 Create and Send co-planning email templates facilitate
- [ ] #1892 Email Alerts facilitate

**Comment by staceypark** [comment]
@lijenn @hsnyder2 @apisano0920 

Sharepoint Set-up in the context of Dynamics 365 - to have live caching.

Link to a location that can be live edited.
Version control...transparency for the team.
Sharepoint is the "catch all" on the backend for 365 apps, including Dynamics.
The team starts being set up at the co-planning phase in D365, right now, we also create a Teams group, so they have a channel for: 
- All their team meetings are scheduled from this channel.
- All their do cuments are here.

So, is Dynamics how we will set up Team channels for their team meetings and files, including co-planning documents.
IDEALLY, YES.

Following the O.H.I.O. method, when set up a group in Dynamics, it would create a Teams group. 

_Has there been any progress on Teams integration with Dynamics? 
Are there any ways that creating groups in Dynamics would be more likely to translate to Teams groups, if/when integration between Dynamics and Teams becomes available?_

@staceypark will show our current process by which a team's SharePoint is setup during next Wed 6/2's CRM meeting

**Comment by lijenn** [comment]
Thanks @staceypark!

@hsnyder2 @apisano0920 
Some more context and the manual workflow on how we use the Microsoft Teams with the Learner teams before Wed. 6/2 meeting:
- Each _MTL_ Team gets their own Microsoft Teams team with the meeting link generated in a channel:
![image](https://user-images.githubusercontent.com/59668647/120051705-84fcd200-bfd6-11eb-88e2-f0228bd06452.png)
  - Note: We do not add people into the auto-created Microsoft Teams meeting invite because we want the invite to come from MTL Help, see next bullet. We only take and copy the meeting link that is generated once setting up a meeting in the channel.
- Meeting link is copied and pasted to an Meeting Appointment from the MTL Help Outlook Calendar to send the meeting invites out
![image](https://user-images.githubusercontent.com/59668647/120051613-22a3d180-bfd6-11eb-83ae-fbd7f6efc423.png)
- We include the co-planning files/documents in the the Files tab in their Microsoft Teams for the Learners and Facilitators to refer to at any time
--------------------------------------------------------
Regarding the **bulk upload/OHIO set up**:
Since Microsoft Teams, SharePoint, and MS 365 Groups are all interconnected, this handles our Only Handle It Once set up and permissioning. 
This is the same idea we talked about on Wednesday regarding the idea of the excel file to set up Learners in the Sim:
(FYI @jamesmrollins We may have an updated version of the excel file to test for the bulk load of setting up teams in the Sim UI that is exported from Dynamics)
1. **Microsoft Teams**: We create a Microsoft Teams team and add the members in it. We use this platform for the team meetings, track the Chat and overall discussion, and refer back to co-planning files for the Learner and Facilitators.
2. **SharePoint**: A SharePoint site is then auto-created and the files added to Microsoft Teams are automatically stored on the cloud --> live caching/can share and edit files with multiple people through a link.
- We have been modifying the SharePoint page to our style:
![image](https://user-images.githubusercontent.com/59668647/120052311-95fb1280-bfd9-11eb-858f-f18a8120bd05.png)
3. **MS 365 Group**: A MS 365 group is auto created and generates a group email (acts like a distribution email) which we have been using in our pre/post emails and general emails to the MTL teams. We had also looked into this due to the possibility of bulk permissioning for Story 5. 
- This is accessed via the Azure Portal in the Active Directory Admin Center, I believe there is a way to view these via Outlook, but I have not figured that out yet or do not have the right permissions.
![image](https://user-images.githubusercontent.com/59668647/120052541-a8298080-bfda-11eb-94b2-dc05aa17a395.png)
- @staceypark @anazariz **For Story 5,** Anthony did a quick check to see if using the MS 365 Group emails could be used when trying to give folder permission on the new SharePoint site for mtl.how/data, at a glance it looked like the permission field accepted the group email. However, I recommend we still through a full QA test with it while testing the data files and folder permissions for Story 5. 
   - Note: The email that is generated follows the name of the Microsoft Teams team, renaming the Microsoft Teams team does not change the email that is created. 
   @staceypark That might be of concern if we're thinking about the most optimal state that we have discussed to have Microsoft Teams team for the Co-planning phase and then editing the title and transition it to be in the Team Meeting phase since the MS 365 email might not reflect the MTL team's name.
5. **Dynamics**: See the 2 italicized questions from Stacey's post above.

**Comment by staceypark** [comment]
Update from 6/2 d365 meeting:
@hsnyder2 @apisano0920 have not heard any updates regarding the Teams & Dynamics integration and feel that it is not possible to create a solution that is set up to integrate better if/when the integration happens as they are not able to predict what it will look like.

They are still working on the Outlook integration piece and we will be able to create calendar invitations from Dynamics through the MTL Help & Info emails. But we would not be able to create the Teams.

**Comment by anazariz** [comment]
@lijenn, do you still want to keep this card open for reference?
